src/tables/STAT.js

```diff
@@ -0,0 +1,54 @@
+import r from 'restructure';
+
+
+let AxisRecord = new r.Struct({
+  axisTag:      new r.String(4),    // e.g. 'ital', 'wght'
+  axisNameID:   r.uint16,
+  axisOrdering: r.uint16,
+})
```

Comment:
Oh frabjous day.  Attempting to check for missing semicolons also in the test file, to copy styles seen in other test files, eslint just kinda mentioned that I wasn't actually using a couple of the binary data sample tables 02, 03, 04. Missing semicolons and missed tests verified.  Other cleanups done.